ui/shared/components/panel/variants/Annotations.jsx

```diff
@@ -117,11 +162,15 @@ const BaseSearchLinks = React.memo(({ variant, mainTranscript, genesById }) => {
 
   const seqrLinkProps = { genomeVersion: variant.genomeVersion, svType: variant.svType }
   if (variant.svType) {
-    seqrLinkProps.location = `${variant.chrom}:${variant.pos}-${variant.end}%20`
+    if (variant.endChrom && variant.endChrom !== variant.chrom) {
+      seqrLinkProps.location = `${variant.chrom}:${variant.pos - 50}-${variant.pos + 50} ${variant.endChrom}:${variant.end - 50}-${variant.end + 50}`
+    } else {
+      seqrLinkProps.location = `${variant.chrom}:${variant.pos}-${variant.end}%20`
```

Comment:
I'm not very familiar with the location property. Does it require a '%20' at the end? If yes, then a '%20' needs to be added to the end of the first assignment.

Reply:
this is setting the location parameter passed to search if you are searching for all of the "same" variants as this one. The search already only includes variants of the same SV type, and if there is no endChrom it maintains current behavior of returning variants where both their start and end are within 20% of the overall SV size of the respective start and end of this variant. In the case of endChrom, this will now instead search for variants (of the same SV type) where their start position is either within 50bp of the start or end of the variant. For endChrom variants, the %20 is not relevant, as the breakpoints are not within the same chromosome and the size is functionally 0 so using a size percentage is a bad idea. However, thinking about this, for translocations/ cpx variants in multiple chromosomes, I think we actually want to return anything where the start is nearby and not worry about filtering against the end. I will change the logic to just be within a 50bp window of the start position for now, and can work with the analysts later if it is not working well for them

Reply:
That's helpful. I thought the '%20' was a blank char.